### DISCUSSION TITLE
An error message in the convesation flow

### DISCUSSION THREAD

**Discussion by tokshiva** [FlowiseAI/Flowise#619] [Q&A]
Hi, I'm new to flowise. I built a conversation flow by integrating openAI API+pinecone+flowise.
I get the following error message on flowise when I started the conversation.
Kindly clarify why this error occurs.

InsufficientQuotaError: You exceeded your current quota, please check your plan and billing details.

**Comment by chungyau97** [comment]
If the error code is 429 you could check out [OpenAI Usage](https://platform.openai.com/account/usage)
